lib/views/login_register/login.dart

```diff
@@ -1,11 +1,48 @@
 import 'package:flutter/material.dart';
-import 'package:thc/models/bloc.dart';
+import 'package:thc/models/navigator.dart';
```

Comment:
The global key & extension type in the `navigator` file makes those navigation calls a lot cleaner. :)